Classes/WPEditorView.m

```diff
@@ -27,13 +27,11 @@
 
 static NSString* const WPEditorViewWebViewContentSizeKey = @"contentSize";
 
-static NSInteger CaretPositionUnknow = -9999;
-
 @interface WPEditorView () <UITextViewDelegate, UIWebViewDelegate, UITextFieldDelegate>
 
 #pragma mark - Cached caret & line data
-@property (nonatomic, assign, readwrite) CGFloat caretYOffset;
-@property (nonatomic, assign, readwrite) CGFloat lineHeight;
+@property (nonatomic, copy, readwrite) NSNumber *caretYOffset;
+@property (nonatomic, copy, readwrite) NSNumber *lineHeight;
```

Comment:
Well, `copy` basically ensures that these properties are not changed after being assigned, but in this case there's no such risk, since the original objects are also handled entirely by this class.

Will change these.